docs/docs/marker.mdx

```diff
@@ -0,0 +1,280 @@
+    - action: provide_carbon_estimate
+```
+
+And here is a marker for surfacing dialogues where all mandatory slots are set except `travel_flight_class`:
+
+```yaml
+marker_dialogue_completion:
+  and:
+    - slot_was_set: travel_departure
+	  - slot_was_set: travel_destination
+  	- not:
+			slot_was_set: travel_flight_class
+```
+
+## Defining Markers
+
+To describe the events that we're interested in, we provide `conditions` and `operators`.
+
+Conditions are simple expressions that describe an event, and are used like to variables in a logical expression, for example `action: utter_greet`. Operators allow you to combine conditions or sub-marker definitions which include nested operators and conditions.
+
+### Conditions
+
+We support the following conditions:
+
+- `action`: the specified bot action was executed.
+- `intent`: the specified user intent was detected.
+- `slot_was_set`: the specified slot was set.
+
+We also support their negated forms:
+
+- `not_action`: the event is not an `ActionExecuted` with the specified action.
+- `not_intent`: the event is not a `UserUttered` with the specified intent.
+- `slot_was_not_set`: the specified slot has not been set.
+
+### Operators
+
+To combine conditions, we provide the following operators:
+
+- `and`: all sub-conditions applied.
+- `or`: any of the sub-conditions applied.
+- `not`: the sub-condition did not apply. `not` can have only 1 condition.
+- `seq`: the list of sub-conditions applied in the specified order with any number of events occurring in-between.
+- `at_least_once`: the listed sub-marker definitions occurred at least once. Only the first occurrence will be marked.
+- `never`: the listed sub-marker definitions never occurred.
+
+### Marker Configuration
+
+To define one or more marker, you need to specify a `yaml` configuration file. Here is an example:
+
+```yaml
+marker_name_provided:
+  slot_was_set: name
+marker_mood_expressed:
+  or:
+    - intent: mood_unhappy
+    - intent: mood_great
+marker_cheer_up:
+  seq:
+    - intent: mood_unhappy
+    - action: utter_cheer_up
+marker_bot_challenged:
+  at_least_once:
+    - intent: bot_challenge
+marker_mood_expressed_and_name_provided:
+  and:
+		- slot_was_set: name
+    - or:
+      - intent: mood_unhappy
+      - intent: mood_great
+```
+
+The top level contains unique user-specified names for each marker. This is a dictionary mapping marker names to marker definitions.
+
+Each marker definition can contain an arbitrary combination of `conditions` and `operators` that map to a valid sub-marker definition.
+
+Each `condition` maps a condition tag to a value. For example, in `marker_name_provided`, the tag `slot_was_set` is mapped to the slot `name` which exists in the `domain.yaml` file.
+
+Each `operator` is a dictionary mapping an operator tag (e.g., `and`) to a list of sub markers, which can be conditions or a nested operator-conditions definition (e.g., see `marker_mood_expressed_and_name_provided`).
+
+<aside>
```

Comment:
This seems like it shouldn't be here for markdown but I'm not sure if it's something that's picked up by our docs renderer

Reply:
replaced it with 
```
:::note
text goes here 
:::
```
which I found in other docs.